packages/dso-toolkit/src/components/tile-grid/tile-grid.stories-of.ts

```diff
@@ -1,33 +1,47 @@
-import { TileGridArgs, tileGridArgTypes } from "./tile-grid.args.js";
-import { TileGrid } from "../tile-grid/tile-grid.models.js";
-import { tiles } from "./tile-grid.content.js";
```

Comment:
Waarom heb je deze import hier weggehaald?

Reply:
Deze wordt niet meer gebruikt in de code